src/Backoffice/Task/Domain/Enums/StatusEnum.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Task\Domain\Enums;
+
+enum StatusEnum: string
+{
+    case TODO = 'TO_DO';
+    case INPROGRESS = 'IN_PROGRESS';
+    case ACCEPTED = 'ACCEPTED';
```

Comment:
Please leave capital words for const name only. 

```suggestion
    case TODO = 'to_do';
    case INPROGRESS = 'in_progress';
    case ACCEPTED = 'accepted';
```